core/build.gradle

```diff
@@ -58,6 +58,7 @@ dependencies {
   compile 'org.elasticsearch:securesm:1.1'
 
   // utilities
+  compile "org.elasticsearch:elasticsearch-cli:${version}"
```

Comment:
Do we actually need to publish this? If we first move the ES cli under a `distribution/tools` project, then this could could be something like `distribution/tools/base-cli`?